docs/docusaurus/docs/cloud/set_up_gx_cloud.md

```diff
@@ -4,11 +4,24 @@ title: 'Set up GX Cloud'
 description: Set up GX Cloud.
 ---
 
-To get the most out of GX Cloud, you'll need to request a GX Cloud Beta account, generate and record access tokens, set environment variables, and then install and start the GX Agent.
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
 
-Currently, the GX Cloud user interface is configured for Snowflake. If you don't have a Snowflake Data Source, you won't be able to connect to Data Assets, create Expectation Suites and Expectations, run Validations, or create Checkpoints. 
+If you're the first person in your org to access GX Cloud, you're assigned Admin permissions. You're responsible for generating and recording access tokens, setting environment variables, installing and starting the GX Cloud agent, and inviting users to your org. 
```

Comment:
```suggestion
If you're the first person in your org to access GX Cloud, you're assigned Admin permissions. Admins are responsible for generating and recording access tokens, setting environment variables, installing and starting the GX Cloud agent, and inviting users to the org. 
```
This reads as though the first person to access the org is solely responsible for these tasks. It's worth noting that the first person to access the org will have admin permissions, but they may not be the only one to receive admin permissions (for instance, if they add another user and assign them as an admin).

Reply:
@Rachel-Reverie, this topic no longer appears in the TOC. I am keeping it temporarily in case it is required when additional Data Sources are supported.